pkg/rpc/response/result/invoke.go

```diff
@@ -46,7 +48,7 @@ func (r Invoke) MarshalJSON() ([]byte, error) {
 	}
 	return json.Marshal(&invokeAux{
 		GasConsumed:    r.GasConsumed,
-		Script:         r.Script,
+		Script:         base64.StdEncoding.EncodeToString(r.Script),
```

Comment:
`invokeAux` can have `Script []byte`, as `base64` is the default method of json byte encoding.